Library/Homebrew/keg_only_reason.rb

```diff
@@ -36,7 +42,7 @@ def applicable?
 
   sig { returns(String) }
   def to_s
-    return @explanation unless @explanation.empty?
+    return @explanation if @explanation.present?
```

Comment:
If this never errored before, surely that means the type of `@explanation` should be `String` rather than `T.nilable(String)`?

Reply:
Agreed 👍🏻